services/server/src/routes/chat.ts

```diff
@@ -8,7 +8,7 @@ router.get("/list/rooms/:userId", chat.findUserRooms);
 router.get("/list/:roomId/messages", chat.findRoomMessages);
 router.get("/room/by/members/", chat.findRoomByMembers);
 router.get("/room/members/:roomId", chat.findRoomMembers);
-router.get("/room/call/:call", chat.findCallRoom);
+router.get("/room/session/:sessionId", chat.findSessionRoom);
```

Comment:
We can remove route as well as in the frontend we can disable the api call for this rount. 

We can use the `findRoomByMembers` isntead to do the same function.

Reply:
@mmoehabb this route can be removed